icon-naming-utils is a native dependency so we need to use the native pkg-config
to find it.  Simply perform the old switcheroonie on $PKG_CONFIG so
pkg-config-native is used temporarily.

Upstream-Status: Inappropriate (OE-specific)
Signed-off-by: Ross Burton <ross.burton@intel.com>

diff --git a/configure.ac b/configure.ac
index 6074f2c..3d38e49 100644
--- a/configure.ac
+++ b/configure.ac
@@ -40,6 +40,8 @@ if test "x$enable_mapping" != "xno"; then
    UTILS_REQUIRED=0.8.7
 
    AC_MSG_CHECKING([icon-naming-utils >= $UTILS_REQUIRED])
+   save_PKG_CONFIG=$PKG_CONFIG
+   PKG_CONFIG=pkg-config-native
    PKG_CHECK_EXISTS(icon-naming-utils >= $UTILS_REQUIRED,
                     have_utils=yes, have_utils=no)
    if test "x$have_utils" = "xyes"; then
@@ -51,6 +53,7 @@ if test "x$enable_mapping" != "xno"; then
       AC_MSG_ERROR([icon-naming-utils >= $UTILS_REQUIRED is required to build
                     and install gnome-icon-theme])
    fi
+   PKG_CONFIG=save_PKG_CONFIG
 else
    ICONMAP="false"
 fi
